ocrd_validators/ocrd_validators/ocrd_tool.schema.yml

```diff
@@ -118,7 +104,6 @@ properties:
                 - recognition/font-identification
                 - recognition/post-correction
                 - layout/segmentation
-                - layout/segmentation/text-nontext
```

Comment:
Why these changes to `ocrd_tool.schema.yml`?

Reply:
Oops, sorry! That was the result of a (failed) `make install`. Revert or amend+force-push, you or me?

Reply:
I would amend/forcepush to keep history clean. If you can do it, that would be great.

Reply:
Done.